Configuration/AlCa/python/autoCond.py

```diff
@@ -20,19 +20,19 @@
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v8',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun2_v12',
+    'run1_data'         :   '80X_dataRun2_v13',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   '80X_dataRun2_v12',
+    'run2_data'         :   '80X_dataRun2_v13',
     # GlobalTag for Run2 data relvals: allows customization to run with fixed L1 menu
-    'run2_data_relval'  :   '80X_dataRun2_relval_v8',
+    'run2_data_relval'  :   '80X_dataRun2_relval_v9',
     # GlobalTag for Run1 HLT: it points to the online GT
-    'run1_hlt'          :   '80X_dataRun2_HLT_frozen_v11',
+    'run1_hlt'          :   '80X_dataRun2_HLT_frozen_v12',
```

Comment:
only HLT GTs are actually changing.
The rest are already in the release, but github shows as if they are changed as well.